dm/pkg/schema/tracker.go

```diff
@@ -440,39 +439,15 @@ func (tr *Tracker) GetDownStreamTableInfo(tctx *tcontext.Context, tableID string
 			return nil, err
 		}
 
-		dti = GetDownStreamTI(downstreamTI, originTi)
+		dti = &DownstreamTableInfo{
+			TableInfo:        downstreamTI,
+			WhereHandleCache: sqlmodel.GetWhereHandle(originTi, downstreamTI),
```

Comment:
maybe can rename `downstreamTI` to `downstreamTi` 😂

Reply:
TI = TableInfo

Reply:
but another one called originTi 😂